workers/unity/Packages/io.improbable.gdk.core/Worker/OpListConverter.cs

```diff
@@ -86,7 +87,9 @@ public bool ParseOpListIntoDiff(OpList opList, CommandMetaData commandMetaData)
                         break;
                     case OpType.AuthorityChange:
                         var authorityOp = opList.GetAuthorityChangeOp(i);
-                        viewDiff.SetAuthority(authorityOp.EntityId, authorityOp.ComponentId, authorityOp.Authority);
+                        viewDiff.SetAuthority(authorityOp.EntityId, authorityOp.ComponentId, authorityOp.Authority,
```

Comment:
Out of curiosity, do we have a standard about using `Function(integer); integer++;` vs `Function(integer++);` since the former is clearer (I usually use the latter)?

Reply:
We don't have a standard for it 😃